paths.csv

```diff
@@ -1,19 +1 @@
D-Link,camera,DCS-2332,(28:10:7B),rtsp://{{ip_address}}/live{{stream_number}}.sdp,H.264,554,1;2;3,,,,3,1,admin,,,http://www.dlink.com/-/media/Consumer_Products/DCS/DCS%202332L/Manual/DCS_2332L_A1_Manual_v100_WW.pdf
FLIR,nvr,DNR200,(90:02:A9),rtsp://{{ip_address}}/cam/realmonitor?channel={{channel_number}}&subtype={{stream_number}}&authbasic={{#base64}}{{username}}:{{password}}{{/base64}},H.264,554,0;1,0;1;2;3;4;5;6;7;8;9;10;11;12;13;14;15,,,,,admin,admin,,http://www.flir.com/uploadedFiles/Security/Products/Security-NVR/DNR200-Series/FLIR-DNR200-Series-Manual-EN.pdf
Axis,camera,M3024-LVE,(AC:CC:8E),rtsp://{{ip_address}}/axis-media/media.amp,H.264,80,,,,,,,root,,,http://www.axis.com/us/en/products/axis-m3024-lve/support-and-documentation
Axis,camera,P1224���E,(00:40:8C),rtsp://{{ip_address}}/axis-media/media.amp,H.264,554,,,,,,,root,,,http://www.axis.com/files/manuals/um_p1224e_57828_en_1405.pdf
Axis,camera,207W,(00:40:8C),rtsp://{{ip_address}}/mpeg4/media.amp,MPEG4,554,,,,,,,root,,TRUE,http://www.axis.com/files/manuals/um_207w_mw_33158_en_1208.pdf
Hikvision,camera,DS-2CD2032-I,(C4:2F:90|C0:56:E3|BC:AD:28|44:19:B6|28:57:BE),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/PSIA/streaming/channels/10{{stream_number}},H.264,554,1;2,,,,2,1,admin,,,http://www.hikvision.com/UploadFile/image/2013061411054615328.pdf
Digital Watchdog,nvr,VMAX A1,(00:1C:84|3C:04:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/rtsp/unicast/live/profile-{{channel_number}},H.264,554,,1;2;3;4,,,,,admin,,,http://digital-watchdog.com/forcedown?file_path=_gendownloads/fe8bcd0f-58df-4414-ae3c-bc76fda207c1/&file_name=DW_MAN_VMAX%20A1_Rev0416.pdf
Digital Watchdog,nvr,VMAX 960H Core,(00:1C:84|3C:04:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/rtsp/unicast/live/profile-{{channel_number}},H.264,554,,1;2;3;4,,,,,admin,,,http://www.advancealarms.com/user-manuals/digital-watchdog-vmax-960h-core.pdf
Digital Watchdog,nvr,VMAX IP Plus,(00:1C:84|3C:04:BF),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/rtsp/unicast/live/profile-{{channel_number}},H.264,554,,1;2;3;4;5;6;7;8;9;10;11;12;13;14;15;16,,,,,admin,,,http://digital-watchdog.com/forcedown?file_path=_gendownloads/1a4b1b60-fee9-4212-9229-5c3e27685431/&file_name=DW_MAN_VMAXIPPlus_Rev0416.pdf
Amcrest,nvr,NV1104,(90:02:A9),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel_number}}&subtype={{stream_number}},H.264,554,0;1,0;1;2;3,,,,,admin,admin,,http://amcrest.com/fileuploader/download/download/?d=0&file=custom%2Fupload%2FAmcrest_NVR_User_Manual_V3.3.pdf
Amcrest,camera,IP2M-841,(90:02:A9),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/cam/realmonitor?channel={{channel_number}}&subtype={{stream_number}},H.264,554,0;1,,,,1,0,admin,admin,,https://support.amcrest.com/hc/en-us/article_attachments/203939917/Amcrest_PT_Camera_User_Manual_v1.1.7.pdf
```

Comment:
Since it's not numeric and instead an enumerated list of values, I think we need `channel` and `stream` instead of `channel_number` and `stream_number`. 

So it should be `low_res_stream` and `high_res_stream` too.

Also, there's odd character encoding after P1224, so check that your using UTF-8 character encoding for this file.